examples/03_turbine_ideal_energy.ipynb

```diff
@@ -0,0 +1,405 @@
+      "INFO:project_ENGIE:Converting field names to IEC 61400-25 standard\n",
+      "INFO:operational_analysis.types.timeseries_table:Loading name:plant_data\n",
+      "INFO:operational_analysis.types.timeseries_table:Loading name:plant_data\n",
+      "INFO:operational_analysis.types.timeseries_table:Loading name:merra2_la_haute_borne\n",
+      "INFO:operational_analysis.types.timeseries_table:Loading name:era5_wind_la_haute_borne\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Prepare data\n",
+    "project.prepare()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "Index(['id', 'wrot_BlPthAngVal1_avg', 'wmet_wdspd_avg', 'wmet_VaneDir_avg',\n",
+       "       'wmet_EnvTmp_avg', 'wyaw_YwAng_avg', 'wmet_HorWdDir_avg', 'wtur_W_avg',\n",
+       "       'energy_kwh'],\n",
+       "      dtype='object')"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "project._scada.df.columns"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "INFO:operational_analysis.methods.turbine_long_term_gross_energy:Initializing TurbineLongTermGrossEnergy Object\n",
+      "INFO:operational_analysis.methods.turbine_long_term_gross_energy:Processing SCADA data into dictionaries by turbine (this can take a while)\n"
+     ]
+    }
+   ],
+   "source": [
+    "ta = turbine_long_term_gross_energy.TurbineLongTermGrossEnergy(project)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "INFO:operational_analysis.methods.turbine_long_term_gross_energy:Filtering turbine data\n",
+      "INFO:operational_analysis.methods.turbine_long_term_gross_energy:Processing reanalysis data to daily averages\n",
+      "INFO:operational_analysis.methods.turbine_long_term_gross_energy:Processing scada data to daily sums\n",
+      "4it [00:00, 24.56it/s]\n",
+      "INFO:operational_analysis.methods.turbine_long_term_gross_energy:Setting up daily data for model fitting\n",
+      "INFO:operational_analysis.methods.turbine_long_term_gross_energy:Fitting model data\n",
+      "INFO:operational_analysis.methods.turbine_long_term_gross_energy:Applying fitting results to calculate long-term gross energy\n",
+      "INFO:operational_analysis.methods.turbine_long_term_gross_energy:Run completed\n"
+     ]
+    }
+   ],
+   "source": [
+    "ta.run(reanal_subset = ['era5', 'merra2'], \n",
+    "       max_power_filter = 0.85, \n",
+    "       wind_bin_thresh = 2, \n",
+    "       correction_threshold = 0.90, \n",
+    "       enable_plotting = False,\n",
+    "       plot_dir = None)"
```

Comment:
PR #77 now allows filtering parameters to be passed as arguments in the UQ = False.
Should I also move all the arguments back to run instead of init to make the new version backward compatible?

Reply:
PR #77 should fix this.

Reply:
I can get 03_notebook to work by modifying the inputs in the init and run functions. Is PR 77 still needed?

Reply:
PR 77 does fix some other things too (like hardcoded parameters that are changed to arguments), so let's keep it.